internal/endorser/endorser.go

```diff
@@ -73,22 +83,35 @@ func verifyAndSummarizeProvenances(referenceValues *verification.ReferenceValues
 		provenancesData = append(provenancesData, p.SourceMetadata)
 	}
 
-	errs := multierr.Append(verifyConsistency(provenanceIRs), verifyProvenances(referenceValues, provenanceIRs))
+	var errs error
+	if len(provenanceIRs) > 0 {
```

Comment:
I am trying to understand what happens if we have multiple provenances but `SkipProvenanceVerification`.

IIUC the control flow lands here, but then `GetReferenceProvenance` would be nil? Is this okay with `verifyProvenances`?

On a higher level: What do we expect to happen then?